mapie/subsample.py

```diff
@@ -74,9 +75,12 @@ def split(
             The testing set indices for that split.
         """
         indices = np.arange(_num_samples(X))
-        n_samples = (
-            self.n_samples if self.n_samples is not None else len(indices)
-        )
+        if self.n_samples is None:
+            n_samples = len(indices)
+        elif isinstance(self.n_samples, float) and 0 < self.n_samples < 1:
```

Comment:
I think this should raise an error no? Say it's 1.2?

Reply:
I agree with this comment. Create a `check_n_samples_ratio` function in utils.py and create unit tests associated.